src/test/java/ImplementAddCandidate.java

```diff
@@ -0,0 +1,41 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.Test;
+
+import java.util.concurrent.TimeUnit;
+
+public class ImplementAddCandidate {
+    @Test
+    public void addCandidate() {
+        WebDriver driver = new ChromeDriver();
```

Comment:
create a `WebDriver` class variable and use it for the whole class

`private WebDriver driver;`